### DISCUSSION TITLE
Why does Hyas have a dependency on bootstrap?

### DISCUSSION THREAD

**Discussion by cbrands** [gethyas/hyas#172] [Q&A]
I understand that Doks has a dependency on bootstrap but why does hyas?

I see bootstrap in package.json, but don't understand the reason.

**Comment by h-enk** [comment]
That's because I prefer to use Bootstrap as default frontend framework.

However, I've been thinking about providing Hyas without any frontend framework at all. And, to optionally use [Hyas CLI](https://github.com/h-enk/hyas-cli) for adding a frontend framework (something like [roots/sage-installer](https://github.com/roots/sage-installer)).
